boards/components/src/process_console.rs

```diff
@@ -7,47 +7,56 @@
 //! Usage
 //! -----
 //! ```rust
-//! let pconsole = ProcessConsoleComponent::new(board_kernel, uart_mux).finalize(());
+//! let pconsole = ProcessConsoleComponent::new(board_kernel, uart_mux, alarm_mux, process_printer)
+//!     .finalize(process_console_component_static!());
 //! ```
 
 // Author: Philip Levis <pal@cs.stanford.edu>
 // Last modified: 6/20/2018
```

Comment:
Do we not update the modified date for changes like this?

Reply:
I don't feel the need, this update is tedious enough.